mkdocs.yml

```diff
@@ -114,7 +114,7 @@ nav:
         - Permission system: permission_management/permission_system.md
         - Work with permissions: permission_management/work_with_permissions.md
         - Manage users: user_management/manage_users.md
-        - Recent activity: recent_activity/recent_activity.md
+        - Review recent activity: recent_activity/recent_activity.md
```

Comment:
Recent activity is a part of dashboad per Radek request to be added in the user profile section, not hidden in Users and permissions.

Reply:
There are three places in user doc where we mention the activity log,
The profile, aimed at mr Smith the editor, few options, basically for seeing ones own actions..
The dashboard - more complex, but in block reference we tend not to describe functionality in full, due to consistency, space consstraints, etc.
Admin UI - most developed, aimed at the admin persona, with filtering by users, types of events, etc. This is where a robust description should go IMVHO
@radzyu ?

Reply:
@juskora 
Originally if I didn't describe so much the dashboard or the profile recent activity it's because there juste embedments of the main feature, the admin page. The 3 elements are protected by the same permission.